README.md

```diff
@@ -128,7 +128,15 @@ In Browser, add a line to your index.html or equivalent BEFORE including the bun
 ```
 <script type="text/javascript">localStorage.debug = "dweb-archive dweb-archive:* dweb-transports dweb-transports:* dweb-objects dweb-objects:*";</script>
 ```
-
+## Contributing
+* Create a new branch with a meaningful name `git checkout -b branch_name`.<br />
+* Add the files you changed `git add file_name`.<br />
+* Commit your changes `git commit -m "Message briefly explaining the feature"`.<br />
+* Keep one commit per feature. If you forgot to add changes, you can edit the previous commit `git commit --amend`.<br />
+* Push to your repo `git push --set-upstream origin branch-name`.<br />
+* Go into [the Github repo](https://github.com/internetarchive/iaux.git) and create a pull request explaining your changes.<br />
+* If you are requested to make changes, edit your commit using `git commit --amend`, push again and the pull request will edit automatically.<br />
+* If the PR is related to any front end change, please attach relevant screenshots in the pull request description.<br/>
```

Comment:
This is requested in the pull request template, so there's no need to explain that portion of the pull request process here.